drivers/ir_nec/ir_nec.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2022 Dario Petrillo
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ir_nec
+ * @{
+ *
+ * @file
+ * @brief       Device driver implementation for the IR NEC Remote receiver
+ *
+ * @author      Dario Petrillo <dario.pk1@gmail.com>
+ *
+ * @}
+ */
+
+#include "ir_nec.h"
+#include "ir_nec_constants.h"
+#include "ir_nec_params.h"
+
+#include <stdlib.h>
+#include "xtimer.h"
+
+#include "ir_nec.h"
+
+static void ir_nec_isr(void *arg)
+{
+    ir_nec_t *ir = (ir_nec_t *)arg;
+
+    if(gpio_read(ir->pin) != 0) // rising edge
```

Comment:
the coding convetion of RIOT is : `if ( ) {` space after if and curly-brace in the same line (for every block not function).